composer.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "utopia-php/fetch",
+    "description": "A simple library that provides an interface for making HTTP Requests.",
+    "type": "library",
+    "license": "MIT",
+    "require-dev": {
+        "phpunit/phpunit": "^9.5",
+        "laravel/pint": "*",
```

```suggestion
        "laravel/pint": "^1.5.0",
```

its the latest version that support PHP 8.0, we are not able to go further yet 👍🏻